src/Structures/NezuGateway.ts

```diff
@@ -55,8 +55,8 @@ export class NezuGateway extends EventEmitter {
             afk: false
         },
         updateSessionInfo: async (shardId: number, sessionInfo: SessionInfo | null) => {
-            if (gatewayResume && sessionInfo !== null) {
-                await this.drizzle.insert(schema.sessions).values({
+            await (gatewayResume && sessionInfo !== null
```

Comment:
Why the ternary?

Reply:
this is because of ur eslint :) it force me to use ternary